templates/zerver/help/change-a-user's-name.md

```diff
@@ -0,0 +1,25 @@
+# Change a user's name
+Name of the user can be changed by the admin as well as user himself.
```

Comment:
I would rephrase this as "An admin can change the name of another user.  Ordinary users can change their own names, so admins should generally try to teach users how to do this for themselves, but admins can do it when users are unavailable."